vql/linux/audit/audit_listener.go

```diff
@@ -79,6 +73,10 @@ func openEpollDescriptor(fd int) (int, error) {
 }
 
 func (self *AuditListener) Open(ctx context.Context) error {
```

Comment:
`ctx` not used

Reply:
It is used in the test listener, though, to create a nested context that is used to cancel Wait().